### DISCUSSION TITLE
Getting individual elements from the JSON object

### DISCUSSION THREAD

**Discussion by jessecheu** [alexzaganelli/strapi-plugin-email-designer#44] [Q&A] [answered]
Hi, you have the example 
```
{{ _.forEach(order.products, function(product) { }}
	<li>{{- product.name }}</li>
	<li>{{- product.price }}</li>
{{ }); }}
```
for rendering an array of objects. 
How can I render just a single attribute from the object like USER.firstname in the example?
```
      {
        // this object must include all variables you're using in your email template
        USER: {
          firstname: 'John',
          lastname: 'Doe',
        },
        order: {
          products: [
            { name: 'Article 1', price: 9.99 },
            { name: 'Article 2', price: 5.55 },
          ],
        },
        shippingCost: 5,
        total: 20.54,
      }
```

I have never used lodash so this is the first time I am having to try it. 
I noticed from the lodash document it would be _.get(), I tried
` {{  _.get(orderDetails, 'user.firstname', 'Guest'); }}.`
Any suggestions would be helpful, thank you.

**Comment by alexzaganelli** [comment, marked as answer]
Hi @jessecdob, you should be able to access to object keys using the dot notation or the bracket notation like this:

```javascript
// dot notation
{{= USER.firstname }}

// or bracket notation

{{= USER["firstname"] }}

// or lodash .get method

{{=  _.get(USER, 'firstname', 'Guest'); }}
```

I think you missed the ```=``` sign after the parenthesis, which is necessary to interpolate the code. Here the [docs](https://lodash.com/docs/4.17.15#template) 

Ps. do you like the plugin?

**Reply by jessecheu** [reply]
Great, Thank you. The Plugin has been a blessing.
